playground/mel_playground.cppo.ml

```diff
@@ -186,7 +186,7 @@ let compile =
     let types_signature = ref [] in
     warnings_collected := [];
     try
-      let lexbuf = Lexing.from_string str in
+      let lexbuf = Toolchain.feed_string_with_newline str in
```

Comment:
Ah, so I understand this was the issue, right? We would add the newline when using the toolchain as a library, but not on the playground itself. Great finding!

Reply:
yup!